src/blocks/common.blocks/block-example/block-example.js

```diff
@@ -32,6 +32,15 @@ BEMDOM.decl('block-example', {
                     }
                 });
 
+                Dropdown.on(this.elem('qr'), { modName: 'js', modVal: 'inited' }, function (e) {
```

Comment:
JFI: Such handlers will not be removed by the `i-bem.js` automatically, in case `block-example` will be removed. You must remote it explicitly in `onSetMod → 'js' → ''`